proposed/version-support.md

```diff
@@ -0,0 +1,111 @@
+## Introduction
+
+PHP has a long history of slow adoption of new versions of the language. A major
+factor in that slow adoption curve is the reluctance of public web hosts to upgrade
+their offerings without pressure from their customers, while major PHP projects
+are reluctant to adopt new PHP versions while few public hosts support it. Few
+projects wish to be the first to drop old versions for fear of alienating their
+user base. This chicken-and-egg problem is one of the reasons why many PHP projects
+lag far behind PHP itself in terms of the languages features they adopt.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119][].
+
+[RFC 2119]: http://tools.ietf.org/html/rfc2119
+
+## Goal
+
+This standard serves as a "community covenant" for projects and hosts to agree on
+an approximate schedule for when they will adopt and drop support for PHP versions.
+By synchronizing release schedules, no project is overly penalized for being first
+to drop support for a given PHP version and public hosts have a clear timeline
+they can factor into their normal upgrade schedules.
+
+## Definitions
+
+* **Major version** - A major version of a project is a release that is documented
```

Comment:
I deliberately didn't reference SemVer, as not all projects use semver numbering.  Drupal, for instance, didn't pre-8.0.  (It basically skipped the Minor release.)